Fleece/Core/SharedKeys.hh

```diff
@@ -106,6 +106,8 @@ namespace fleece { namespace impl {
 
         bool isUnknownKey(int key) const FLPURE;
 
+        virtual bool isInTransaction() const FLPURE     {return _inTransaction;}
```

Comment:
Remove `virtual` since this isn't overridden anywhere.

Reply:
Oh yeah, I initial thought I would have to override it in `PersistentSharedKeys`. Done.